pytensor/graph/basic.py

```diff
@@ -558,6 +558,46 @@ def get_parents(self):
             return [self.owner]
         return []
 
+    def convert_string_keys_to_pytensor_variables(self, inputs_to_values):
```

Comment:
Let's make this an internal function of `eval`, no need to have it accessible to other functions in an outer scope

Reply:
Ok then I will insert logic within the implementation part only.